content/rs/installing-upgrading/upgrading.md

```diff
@@ -56,13 +56,10 @@ $ sudo ./install.sh
 
 {{% /note %}}
 
-During the node upgrade process, the services running RS are restarted.
-This will result in a very short interruption to the proxy and node
-itself and the databases running on it. This can be dramatically reduced
-by using multiple active proxies for your database. The client will lose
-the connection to the proxy on the node being upgraded, but then
-automatically reconnect to another active proxy you have configured for
-the database.
+The node upgrade process restarts the services running RS, which causes
+a short interruption to connections to the proxy, node and databases.
+Clusters with [multiple active proxies]({{< relref "/rs/administering/designing-production/networking/multiple-active-proxy.md" >}})
+have reduced interruption time.
```

Comment:
@Paz-Yanover @GitaShmueli What do you think of this?

Reply:
After discussing it f2f with @GitaShmueli  and Miki, and based on their Support experience, let's accept Gita's PR and remove the multi-proxy mention at all.